### ISSUE TITLE
Jest: Unexpected token export

### ISSUE THREAD

**Issue by NillDk** [wardbell/subsink#6]
I have an Angular application with Jest testing.

If I install SubSink as an npm package (https://www.npmjs.com/package/subsink), my tests fail with the following error:

Jest encountered an unexpected token
SyntaxError: Unexpected token export

As far as I understand, this happens because the package has been transpiled to ES2015 and not ES5. I haven't been able to find a Jests configuration solution.

![2019-09-10_1028](https://user-images.githubusercontent.com/435201/64598082-6b0cfd00-d3b7-11e9-8c9b-bc260dcbdabe.png)

**Comment by schuchard** [comment]
have you tried to [Adjust your transformIgnorePatterns whitelist](https://github.com/thymikee/jest-preset-angular#adjust-your-transformignorepatterns-whitelist)

Something like this wherever your jest config is (`jest.config.js` or `package.json`). Below is a `package.json` example.
```json
{
  "jest": {
    "transformIgnorePatterns": [
      "node_modules/(?!subsink)"
    ]
  }
}
```

**Comment by NillDk** [comment]
Yep, I tried that but couldn't make it work.
For now, I have just copied the code into my own repo.

**Comment by kgdiem** [comment]
@NillDk -- you can install my fork from npm, or create your own and update the `module` entry in tsconfig.json, to solve the issue until my PR is merged. 

`npm install --save kgdiem/subsink#master`

**Comment by NillDk** [comment]
@kgdiem - Thanks, I'll take a look at it.
Looking forward to the merge :)

**Comment by ajaynarang** [comment]
> @kgdiem - Thanks, I'll take a look at it.
> Looking forward to the merge :)

Were you able to make it work with this fork?

**Comment by NillDk** [comment]
I actually haven't tried it yet.
I had already copied the code into my own repository and got caught away with other tasks.

**Comment by ajaynarang** [comment]
@NillDk @kgdiem  - Thanks, I just tried it, it worked for me, will check if this is PR is merged. All other solution like adding that to transformIgnorePatterns both in global and local/lib jest.config file didn't work.

**Comment by lendog75** [comment]
Any update on merging this?  I am also have the above issue

**Comment by kgdiem** [comment]
> @NillDk @kgdiem - Thanks, I just tried it, it worked for me, will check if this is PR is merged. All other solution like adding path to transformIgnorePatterns both in global and local/lib jest.config file didn't work.

Yep, my fork is installed and working in my app. Unfortunately there has been no activity in this repo for ~7 months now. 

@wardbell  are you looking for new maintainers?

**Comment by gs-akhan** [comment]
@wardbell  Please merge this.  please

**Comment by wardbell** [comment]
Looking

**Comment by wardbell** [comment]
Closed by PR #8 which really was merged. Thanks
